components/styled/input/src/atoms/toggle.tsx

```diff
@@ -52,13 +57,14 @@ const ToggleInput: FC<ToggleInputProps> = ({ checked, onChange, disabled, ...res
       disabled={disabled}
       onChange={(e) => !disabled && onChange && onChange(e)}
       {...rest}
+      title={checked ? 'On' : 'Off'}
     />
   </StyledToggleWrapper>
 )
 
 interface ToggleInputProps extends HTMLAttributes<HTMLInputElement> {
   checked: boolean
-  disabled?: boolean
+  disabled?: boolean | undefined

```

Comment:
Is there a reason this property still needs to be nullable if its type can now be undefined?

Reply:
Well spotted, cheers. Resolved.